src/test/java/uk/gov/companieshouse/items/orders/api/controller/CertificateItemsControllerUnitTest.java

```diff
@@ -31,48 +36,64 @@
     @Autowired
     private ObjectMapper objectMapper;
 
+    @Autowired
+    private CertificateItemRepository repository;
+
+    private static final String EXPECTED_ITEM_ID = "CHS00000000000000001";
+
     @Test
     @DisplayName("Create creates certificate item")
     void createCertificateItemCreatesCertificateItem() throws Exception {
```

Comment:
I know this hasn't been added in this PR but I would find this easier to understand if I was looking through code if it was simplified to:
```
@DisplayName("Successfully creates certificate item")
void createCertificateItem() throws Exception {
```

Reply:
Good point. Will clarify as you suggest.